src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -159,14 +169,14 @@ public override async Task<dynamic> GetHostNames(string stampName, string siteNa
 
         public override async Task<dynamic> GetSitePostBody(string stampName, string siteName)
         {
-            var response = await Get($"stamps/{stampName}/sites/{siteName}/postbody");
+            var response = await GetObserverResource($"stamps/{stampName}/sites/{siteName}/postbody");
```

Comment:
Is this endpoint configuration going to be wawsobserver-prod? Why don't we use support-bay anymore?

Reply:
We need to remove support-bay-api because it does not exist in national clouds. I can tell you more in person